### ISSUE TITLE
How to remove loaded track ?

### ISSUE THREAD

**Issue by deepaksibar** [igvteam/igv.js#201]
Hi, 
Can we remove single track by passing track name or any id through any external method? As per current implementation igv.Browser.prototype.removeTrack() method need track object to remove track.

So can you please provide me any solution ?

**Comment by skushch** [comment]
```javascript
var removeTrackByName = function (trackName) {
    for (var i = 0, l = igv.browser.trackViews.length; i < l; i++) {
        var trackView = igv.browser.trackViews[i];
        if (trackView.track.name === trackName) {
            igv.browser.removeTrack(trackView.track);
            return;
        }
    }
};
```

**Comment by turner** [comment]
Implemented with commit a20f789b
